protocols/v2/framing-sv2/src/framing2.rs

```diff
@@ -82,20 +89,19 @@ impl<T, B> Default for Sv2Frame<T, B> {
     }
 }
 
+/// Abstraction for a Noise Frame
+/// In practice, it is only used as the alias `HandShakeFrame`
+/// Contains only a `Slice` payload
```

Comment:
I would call it HandShakeFrame and use the alias to rename it NoiseFrame, so we don't break the API but is more explicit what this is

Reply:
I would also add some info about the HandShakeFrame in general. (is not a frame is just an handshake message and have fixed len)

Reply:
https://github.com/stratum-mining/stratum/pull/848/commits/cbafbeae58a160d1a9daec4eb013546f58c261e8